CONTRIBUTING.md

```diff
@@ -2,7 +2,7 @@
 
 This project welcomes contributions and suggestions.  Most contributions require you to agree to a
 Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us
-the rights to use your contribution. For details, visit https://cla.microsoft.com.
+the rights to use your contribution. For details, visit <https://cla.microsoft.com>.
```

Comment:
Why the brackets?

Reply:
Part of the markdown rules are not to have "bare" URLs. Thus, you indicate that the URL is explicitly typed and it should not be text or have an alternative text.